src/messaging-component.js

```diff
@@ -30,7 +30,7 @@ const showExitModal = ({ cartRecovery }) => {
         return false;
     }
     // don't show modal if user has no cart items
-    const cart = JSON.parse(getCookie('paypal-cr-cart') || '{}');
+    const cart = JSON.parse(window.localStorage.getItem('paypal-cr-cart') || '{}');
```

Comment:
existing integrations should have their localStorage populated with existing cart from the cookie instead of `{}`.